distgen.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+
+# before using this script, make sure your image repository contains:
+#   ./templates directory
+#   ./specs/multispec.yml file
+#   ./dg_manifest file
+
+# All templates should be stored in templates directory in container repository
+if [ ! -d "./templates" ]; then
+    echo "Templates for distgen not found"
+    exit
+fi
+
+# expecting file dg_manifest that contains variables:
+#   FILES_TO_COPY in form "path1:mode1 path2:mode2..."
+#   FILES_TO_GEN in form similar to FILES_TO_COPY
+#   INTERNAL_SYMLINKS in form "name1->target1 name1->target1..."
+if [ -f "./dg_manifest" ]; then
+    source ./dg_manifest
```

Comment:
Since manifest file is shell script, could we rename it to `dg_manifest.sh` (or rather remove the `dg_` prefix, as that's not related to distgen) and generate some Makefile snippet from this automatically?   The snippet could be then included.

I sort of miss the possibility to relocate the file (e.g you can not copy/symlink one **source** file onto two places), note that the quite old proposal here https://github.com/devexp-db/cont-postgresql/blob/master/cl-manifest allows this to be specified.